src/Libraries/Microsoft.Extensions.AI.AzureAIInference/AzureAIInferenceExtensions.cs

```diff
@@ -9,6 +9,20 @@ namespace Microsoft.Extensions.AI;
 public static class AzureAIInferenceExtensions
 {
     /// <summary>Gets an <see cref="IChatClient"/> for use with this <see cref="ChatCompletionsClient"/>.</summary>
+    /// <remarks>
+    /// <para>
+    /// No prompt injection defense is performed by default. Applications are expected to implement
+    /// guards against prompt injection attacks as needed.
+    /// </para>
+    /// <para>
+    /// There are no limits or restrictions imposed on data sizes sent to the underlying provider or returned from it
+    /// by default. Applications are expected to apply data size limits as needed.
+    /// </para>
+    /// <para>
+    /// There are no limits or restrictions imposed on the number of messages sent to the underlying provider or returned from
+    /// it by default. Applications are expected to apply rate limits on a per-user or per-session basis as needed.
+    /// </para>
+    /// </remarks>
```

Comment:
This is fine though I wonder if it's really necessary to have it on the `AsIChatClient` methods. It seems like the remark you've added to `IChatClient` already covers it fully, since that's the critical place where the calls occur.

I also like how you phrased it concisely on `IChatClient` better than the longer explanation on each of the `AsIChatClient` methods.

Reply:
Thanks, folks. You've convinced me that the succinct remarks on `IChatClient` are sufficient. I'll keep those and drop the remarks on the implementations.